akka-persistence/src/main/scala/akka/persistence/journal/ReplayFilter.scala

```diff
@@ -46,6 +53,10 @@ private[akka] class ReplayFilter(persistentActor: ActorRef, mode: ReplayFilter.M
   import JournalProtocol._
   import ReplayFilter.{ Warn, Fail, RepairByDiscardOld, Disabled }
 
+  // for binary compatibility
+  def this(persistentActor: ActorRef, mode: ReplayFilter.Mode,
+           windowSize: Int, maxOldWriters: Int) = this(persistentActor, mode, windowSize, maxOldWriters, debugEnabled = false)
+
```

Comment:
I still do not understand. Patrik argued about new `props` method. This is new constructor which is not used anywhere. Why do we need it?